### ISSUE TITLE
Field Group description doesn't display when editing a post

### ISSUE THREAD

**Issue by millionleaves** [pods-framework/pods#6124]
## Description

The Description for a field group does not appear when editing a post with that field group.

## Testing instructions

1. Add a field group and set the description OR set a description on an existing field group.
2. Create/edit an instance of the associated CPT. The description is not visible.

## Expected behavior

Ideally, I'd expect this to appear at the top of the field group since I would like to use it to provide instructions to the content editor.

## Possible workaround

None found. But another option I've seen elsewhere is to provide an HTML "field" type which is used to display content within the field group but does not take input.

## Pods version

2.8 RC

## Other super helpful information

Latest WP, Beaver Builder and Beaver Themer latest.

**Comment by sc0ttkclark** [comment]
I don't think we have the descriptions showing anywhere right now, I'll go and look and see what we should do for this in the mean time. That might be to just disable the description field for now or to add support for a simple `wpautop()` output of the description.

**Comment by sc0ttkclark** [comment]
Going to disable the group description fields for now and treat this as a future enhancement.

**Comment by millionleaves** [comment]
Just a thought - how about adding a pseudo field that doesn't take any input on the front end? This would allow a developr to add more than one to a field group to provide instruction and guidance, and position them between other fields as needed.
